doc/source/examples/unresolved-cfd-dem/single-particle-sedimentation/single-particle-sedimentation.rst

```diff
@@ -0,0 +1,414 @@
+      set particle wall contact force method     = nonlinear
+      set integration method                     = velocity_verlet
+    end
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The gravity is set to 0 as we only need to insert the particles in the specified insertion box.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+      set g                        = 0.0, 0.0, 0.0
+      set number of particle types = 1
+      subsection particle type 0
+        set size distribution type            = uniform
+        set diameter                          = 0.002663
+        set number                            = 1
+        set density particles                 = 1029
+        set young modulus particles           = 1e6
+        set poisson ratio particles           = 0.25
+        set restitution coefficient particles = 0.97
+        set friction coefficient particles    = 0.3
+        set rolling friction particles        = 0.1
+      end
+      set young modulus wall           = 1e6
+      set poisson ratio wall           = 0.25
+      set restitution coefficient wall = 0.97
+      set friction coefficient wall    = 0.3
+      set rolling friction wall        = 0.1
+    end
+
+Insertion Info
+~~~~~~~~~~~~~~~~~~~
+
+We use the list insertion method to insert the single particle in our domain. This is quite convenient, since we can tell exactly what is the position of the particle. We set the insertion frequency to 2000, which is the same as the time end. This way, the particle will be inserted at the top of the channel at the end of the DEM simulation.
+
+.. code-block:: text
+
+    subsection insertion info
+      set insertion method = list
+      set list x           = 0
+      set list y           = 0.08
+      set list z           = 0
+    end
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles initial-particle.prm
+
+.. image:: images/particle_insertion.png
+    :alt: inserted particle at the top of the channel
+    :align: center
+
+After the particle has been inserted it is now possible to simulate its sedimentation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the particle inserted within the previous step. We will discuss the different parameter file sections. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. Most subsections are explained in detail in other CFD-DEM examples such as:  `Gas-solid spouted bed <../../../examples/unresolved-cfd-dem/gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_.
```

Comment:
```suggestion
The CFD simulation is to be carried out using the particle inserted within the previous step. We introduce the different sections of the parameter file ``single-particle-sedimentation.prm`` needed to run this simulation. 
```
You are putting this as the first example, don't refer to a more complex example to explain your simple one!